docs/modules/maintain-cluster/pages/monitoring.adoc

```diff
@@ -1061,6 +1065,8 @@ The following parameters can be used:
 |Health check operation. It can be `all`, `node-state`,
 `cluster-state`, `cluster-safe`, `migration-queue-size` and `cluster-size`.
 
+NOTE: The `clusterSafe` indicator is useful when your cluster is in a passive state. If the cluster is in an active state, the indicator value continually changes due to the dynamic state of the cluster. Also, checking the cluster safety triggers additional operations for each partition and replica, which means operations increase with increased load and may impact cluster performance.
```

Comment:
Maybe "Also, checking the cluster safety triggers additional operations for each partition and replica. Frequent cluster safety checks in a cluster under load may impact cluster performance." ? Basically don't query cluster safety every 5 seconds :)

Reply:
Updated as requested.